.openshiftio/Jenkinsfile.setup.groovy

```diff
@@ -0,0 +1,10 @@
+
+setupEnvironmentPre(env) {
+  sh "if ! oc get -n ${env} configmap app-config -o yaml | grep app-config.yml; then oc create -n ${env} configmap app-config --from-file=app-config.yml; fi"
```

Comment:
Where is that `app-config.yml` file located?

Reply:
Ah yes, I see you guys don't have a duplicate of that file. I've changed it to use the resource configmap file from `.openshiftio`. That should work too. (But please make sure)